File: app/src/main/java/vn/loitp/app/activity/function/viewdraghelper/ViewDragHelperActivity.java
package vn.loitp.app.activity.function.viewdraghelper;

import android.os.Bundle;

import com.core.base.BaseFontActivity;

import loitp.basemaster.R;

public class ViewDragHelperActivity extends BaseFontActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

    }

    @Override
    protected boolean setFullScreen() {
        return false;
    }

    @Override
    protected String setTag() {
        return getClass().getSimpleName();
    }

    @Override
    protected int setLayoutResourceId() {
        return R.layout.activity_view_drag_helper;
    }

}

File: app/src/main/java/vn/loitp/app/activity/function/viewdraghelper/ViewDragHelperActivity.kt
package vn.loitp.app.activity.function.viewdraghelper

import com.loitpcore.annotation.IsFullScreen
import com.loitpcore.annotation.LogTag
import com.loitpcore.core.base.BaseFontActivity
import vn.loitp.app.R

@LogTag("ViewDragHelperActivity")
@IsFullScreen(false)
class ViewDragHelperActivity : BaseFontActivity() {

    override fun setLayoutResourceId(): Int {
        return R.layout.activity_func_view_drag_helper
    }
}
